server/controllers/importers/sif.js

```diff
@@ -74,45 +74,52 @@ module.exports = function (sif) {
   });
 
   var genes = [];
-  entries.forEach(function (entry) {
-    if (entry.length && entry[GENE_NAME] && genes.indexOf(entry[GENE_NAME]) == constants.NOT_FOUND) {
-      genes.push(entry[GENE_NAME]);
+  var links = [];
+  var networkType = "unweighted";
+
+  if (entries.includes(null)) {
```

Comment:
Watch out—`Array.includes` is super new (defined in 2016): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

You can do something like `entries.filter(function (entry) { return entry === null; })` then check if the resulting array is not empty. Not as concise but more backwards-compatible.